pkg/lib/cockpit-components-table.jsx

```diff
@@ -148,6 +175,13 @@ export const ListingTable = ({
             rowProps.onRowClick = (event) => onRowClick(event, row);
         }
 
+        if (rowProps.key && newItems.indexOf(rowProps.key) >= 0) {
+            if (rowProps.className)
+                rowProps.className = rowProps.className + " new-item";
```

Comment:
This is a globally visible class name, and thus should have a name which is less prone to collisions. But see below.

Reply:
Any suggestions? `new-item-animation` or you want to go one step further and add `cockpit` into mix as well e.g. `cockpit-item-animation`?

Reply:
`new-item-ct` seems good enough for now?